tests/test_build_latex.py

```diff
@@ -1416,7 +1416,7 @@ def test_default_latex_documents():
     config.init_values()
     config.add('latex_engine', None, True, None)
     expected = [('index', 'stasi.tex', 'STASI™ Documentation',
-                 r"Wolfgang Schäuble \& G'Beckstein.\@{}", 'manual')]
+                 r"Wolfgang Schäuble \& G\textquotesingle{}Beckstein.\@{}", 'manual')]
```

Comment:
the handling of author by tex escaping seems to be oblivious to smartquotes setting of course as it is not parsed by Docutils I believe. This thus is breaking change because people using  a straight quote in author names saw a curly one in PDF output, but now it will be a straight one (independently of smartquotes setting)

Reply:
+1; Let's do smartquotes in another PR.